serving/docker/partition/properties_manager.py

```diff
@@ -17,20 +17,24 @@
 import requests
 
 # Properties to exclude while generating serving.properties
-from utils import is_engine_mpi_mode, get_engine_configs, get_download_dir, load_properties
+from utils import (is_engine_mpi_mode, get_engine_configs, get_download_dir,
+                   load_properties, update_kwargs_with_env_vars)
 
 EXCLUDE_PROPERTIES = [
     'option.model_id', 'option.save_mp_checkpoint_path', 'model_dir',
     'upload_checkpoints_s3url', 'properties_dir'
 ]
 
+SUPPORTED_QUANTIZATION_METHODS = ["awq"]
+
 
 class PropertiesManager(object):
 
     def __init__(self, args, **kwargs):
         self.entry_point_url = None
         self.properties_dir = args.properties_dir
-        self.properties = load_properties(self.properties_dir)
+        self.properties = update_kwargs_with_env_vars({})
+        self.properties |= (load_properties(self.properties_dir))
```

Comment:
Let's switch the order. Load the properties first and then read from the env variable. That's our flow in model server for inference. Lets do the same for AOT as well